### DISCUSSION TITLE
Exposed livewire code

### DISCUSSION THREAD

**Discussion by leonardyrj** [livewire/livewire#8741] [1. Help]
I need some help with an issue I encountered after launching my project in Laravel 11, implemented with Filament V3 and Livewire 3.

During a pentest, it was identified that the JavaScript framework is not properly compiled. When inspecting the system in Firefox (Debugger tool), I noticed that the Livewire source code is exposed, as shown in the attached image.

What's curious is that this only happens when the environment variable APP_DEBUG is set to false. When APP_DEBUG is set to true, the code is not exposed. From my understanding, it should be the other way around.

I contacted the Filament developers, but they mentioned that they don't have control over this issue. Could this be related to a Livewire configuration?

I need help resolving this matter to ensure the system passes the pentest.

<img width="1129" alt="Captura de Tela 2024-08-10 às 13 01 01" src="https://github.com/user-attachments/assets/23003f14-f39f-4cf1-8abf-eeaed0fb5394">

<img width="1156" alt="Captura de Tela 2024-08-10 às 13 03 15" src="https://github.com/user-attachments/assets/bfcd35ca-7386-4f8b-926b-af9fdf305742">

**Comment by danie-ramdhani** [comment]
Livewire is using minify version when `APP_DEBUG` is `true`, so the file size is much smaller on production mode.

**Comment by leonardyrj** [comment]
But in production we should use APP_DEBUG=false
